### ISSUE TITLE
Are you on a PC or Mac?

### ISSUE THREAD

**Issue by dt-woods** [cga-wm/advgis-beta#12]
Let me know what you have access to as notebooks are now available in ArcGIS Pro v.2.5 (download available through ArcGIS Online via **My Settings** `-->` **Licenses** `-->` and click **Download** next to ArcGIS Pro).

*Yes, the CGA is updating their installations to version 2.5*

**Comment by parthree** [comment]
Yes!  Notebooks are available in Pro.  In the Catalog in your home projects folder you can right-click, then New Notebook, or from the Insert Tab, New Notebook. I'm on a PC.
